warp10/src/main/java/io/warp10/script/WarpScriptLib.java

```diff
@@ -89,6 +89,7 @@
 import io.warp10.script.binary.INPLACEADD;
 import io.warp10.script.binary.LE;
 import io.warp10.script.binary.LT;
+import io.warp10.script.binary.MACROBINARYOP;
```

Comment:
What purpose does this function serve that neither REDUCE nor APPLY provide?

This is the first point to address before we decide we want to add this to the core of WarpScript.